index.html

```diff
@@ -1,19 +1,98 @@
-</head>
-<body>
+    <link
+      href="https://fonts.googleapis.com/css?family=Roboto:400,500,300"
+      rel="stylesheet"
+      type="text/css"
+    />
+    <link rel="stylesheet" href="css/normalize.css" />
+    <link rel="stylesheet" href="css/style.css" />
+    <link rel="shortcut icon" type="image/x-icon" href="favicon.ico" />
+  </head>
+  <body>
+    <!-- NAVBAR -->
+    <header>
+      <nav>
+        <div class="navbar">
+          <img src="img/karma-logo.svg" alt="Logo" />
+
+          <ul class="main-menu">
+            <li class="nav-title"><a href="">Meet Karma</a></li>
+            <li><a href="">How it Works</a></li>
+            <li><a href="">Store</a></li>
+            <li><a href="">Blog</a></li>
+            <li><a href="">Help</a></li>
+            <li><a href="">Login</a></li>
+          </ul>
+        </div>
+      </nav>
+    </header>
+    <!--- END OF NAV BAR -->
+    <section class="cover">
+      <div class="container">
+        <h1>Introducing Karma</h1>
+        <h2>Bring WiFi with you, everywhere you go.</h2>
+        <button>Learn More</button>
+      </div>
+    </section>
+    <!-- PRODUCTS -->
+    <section class="products">
+      <div><h2>Everyone needs a little Karma</h2></div>
+
+      <div class="grid">
+        <div class="box">
+          <div class="box-image">
+            <img src="img/icon-devices.svg" alt="icon showing icon-devices" />
+          </div>
+          <div class="box-text"><p>Internet for all devices</p></div>
+        </div>
+        <div class="box">
+          <div class="box-image">
+            <img src="img/icon-coffee.svg" alt="a coffe cup icon" />
+          </div>
+          <div class="box-text"><p>Boost your productivity</p></div>
+        </div>
+        <div class="box">
+          <div class="box-image">
+            <img src="img/icon-refill.svg" alt="a filling up station icon" />
+          </div>
+          <div class="box-text"><p>Pay as You Go</p></div>
+        </div>
+      </div>
+      <hr />
+    </section>
+    <!---END OF PRODUCTS-->
+
+    <!-- FOOTER -->
+    <footer>
+      <div class="foot-title">
+        <h3>Join us on</h3>
+        <div class="footer-container"></div>
+        <div class="foot-img">
+          <a href=""><img src="img/twitter-icon.svg" alt="Twitter icon" /></a>
+        </div>
+        <div class="foot-img">
+          <a href=""><img src="img/facebook-icon.svg" alt="facebook icon" /></a>
+        </div>
+        <div class="foot-img">
+          <a href=""
+            ><img src="img/instagram-icon.svg" alt="instagram icon"
+          /></a>
+        </div>
+      </div>
+      <div class="foot-text">© Karma Mobility, Inc</div>
+    </footer>
+    <!--END OF FOOTER-->
 
     <!-- Add your HTML markup here -->
     <!-- Remember: Use semantic HTML tags like <header>, <main>, <nav>, <footer>, <section> etc -->
     <!-- All the images you need are in the 'img' folder -->
-
-</body>
+  </body>
```

Comment:
The extra closing body tag is confusing things, please review your first opening body tag to fix this one better,

Reply:
> The extra closing body tag is confusing things, please review your first opening body tag to fix this one better,

Thanks Mathias, I'll make the adjustment. Nice find, Not sure how I managed that one.

Reply:
So i checked through the code on both git and my PC , I don't see the extra body Tag.
I think what you're looking at is the deletion and insertion of the tag from one line to another. hence the red & green highlight.